app/next-client-app/components/data-table/index.tsx

```diff
@@ -168,7 +174,15 @@ export function DataTable<TData, TValue>({
                   colSpan={columns.length}
                   className="h-24 text-center"
                 >
-                  No results.
+                  {emptyStateMessage ? (
```

Comment:
So we shouldn't need any changes to the datatable component now in this PR, so this + the new component props can be deleted